bitnami/redis/README.md

```diff
@@ -388,6 +388,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `metrics.serviceMonitor.interval`            | The interval at which metrics should be scraped                                                  | `30s`                    |
 | `metrics.serviceMonitor.scrapeTimeout`       | The timeout after which the scrape is ended                                                      | `""`                     |
 | `metrics.serviceMonitor.relabellings`        | Metrics relabellings to add to the scrape endpoint                                               | `[]`                     |
+| `metrics.serviceMonitor.metricRelabelings`   | Metrics relabellings to add to the scrape endpoint                                               | `[]`                     |
```

Comment:
Could you please change the description of both relabellings* (just noticed the value has a typo) and metricRelabelings to something that helped differentiate both settings?

From the prometheus-operator CRD: https://github.com/prometheus-operator/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml

```suggestion
| `metrics.serviceMonitor.relabellings`        | Metrics RelabelConfigs to apply to samples before scraping.                                               | `[]`                     |
| `metrics.serviceMonitor.metricRelabelings`   | Metrics RelabelConfigs to apply to samples before ingestion.                                            | `[]`                     |
```